templates/tools/dockerfile/run_tests_python_deps_venv.include

```diff
@@ -0,0 +1,29 @@
+#====================
+# run_tests.py python dependencies
+
+# Basic python dependencies to be able to run tools/run_tests python scripts
+# These dependencies are not sufficient to build gRPC Python, gRPC Python
+# deps are defined elsewhere (e.g. python_deps.include)
+RUN apt-get update && apt-get install -y ${'\\'}
+  python3 ${'\\'}
+  python3-pip ${'\\'}
+  python3-setuptools ${'\\'}
+  python3-yaml ${'\\'}
+  python3-venv ${'\\'}
+  && apt-get clean
+
+# Create a python virtual env for installing required dependencies.
+ENV PYTHON_ENV="/opt/venv/grpc-interop"
+RUN python3 -m venv "$PYTHON_ENV"
+RUN echo "source $PYTHON_ENV/bin/activate" >> ~/.bashrc
+# Use bash explicitly as /bin/sh doesn't support the "source" command.
+# use pinned version of pip to avoid sudden breakages
+# TODO(jtattermusch): currently six is needed for tools/run_tests scripts
```

Comment:
jtattermusch is not in the team anymore. How about we open an issue in the same regard and tag the issue like 

```suggestion
# TODO(issue link): currently six is needed for tools/run_tests scripts
```

We can then find the best assignee for that issue later.

Reply:
Also, the TODO does talk about the usage of python2 is deprecated. Are we good to get rid of `six` dep?

Reply:
Updated the comment. Opened https://github.com/grpc/grpc/issues/36814 to track the cleanup task.

> Are we good to get rid of six dep?

The code test code is still using six, but we should be able to replace its usage with std lib functions.